.gitignore

```diff
@@ -5,3 +5,4 @@ build/
 MANIFEST
 *.egg-info/
 scapy/VERSION
+scapy/contrib/opc_da_ontest.py
```

Comment:
it was just for me when I tested opc da and I didn't want pollute the code with my tests.